website/config/apisix-versions.js

```diff
@@ -0,0 +1,49 @@
+/* eslint-disable quote-props */
+
+/**
+ * @type {Array<string>} version list
+ */
+const versions = [
+  '2.12',
+  '2.13',
+];
+
+/**
+ * @type {Array<{label: string, href: string}>}
+ */
+const archivedVersions = [
+  {
+    label: '2.11',
+    href: 'https://625a9090d04b9a6953165811--2-11-old-docs-apache-apisix.netlify.app/docs/apisix/getting-started/',
```

Comment:
Hi, may I know how to get those links?

Reply:
The site name `2-11-old-docs-apache-apisix.netlify.app` is entered manually. But the `625a9090d04b9a6953165811` is randomly generated. The full host will be returned by netlify cli. 

If the prod attribute is passed in during deployment, it seems that there will be no randomly generated ID. But I haven't tried yet.